McpPlugin/test_eps6800_mcp.ps1

```
@@ -0,0 +1,492 @@
+	$diagnosticScanFlag = -1
+	$diagnosticKeyboardRegisters = @()
+	if ($CaptureDiagnosticScreens -or $CaptureDiagnosticInfo) {
+		$null = Invoke-McpTool 76 "resume"
+		Start-Sleep -Milliseconds 750
+		$null = Invoke-McpTool 77 "request_screenshot"
+		Start-Sleep -Milliseconds 500
+		$diagnosticScreenshot = (Get-ChildItem -LiteralPath $ReleaseDirectory -Filter "screenshot-*.png" |
+			Sort-Object LastWriteTimeUtc | Select-Object -Last 1).FullName
+	}
+	if ($CaptureDiagnosticInfo) {
+		$null = Invoke-McpTool 78 "keyboard_code" @{ code = 0x23; pressed = $true } # 9
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-McpTool' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)